cdrc-rs/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
```

Comment:
are we replacing submodule `cdrc-rs` into an vendored version of it? why?

Reply:
`cdrc-rs` was embedded in the repository during the development of CIRC, and its intention was to facilitate the development process. However, as we have finished it, it will be better to make it a separate submodule.

I have replaced it with the dedicated submodule.